lib/Doctrine/DBAL/Types/DateTimeTzType.php

```diff
@@ -67,6 +67,10 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
      */
     public function convertToDatabaseValue($value, AbstractPlatform $platform)
     {
+        if ($value !== null && !$value instanceof \DateTime) {
```

Comment:
This is actually correct: `null` is an allowed value. Maybe applying De Morgan would make it more readable tho